ws-nextjs-app/pages/[service]/av-embeds/handleAvRoute.ts

```diff
@@ -74,9 +76,17 @@ export default async (context: GetServerSidePropsContext) => {
       isAvEmbeds: true,
       pageData: avEmbed
         ? {
-            ...avEmbed,
             mediaBlock: avEmbed?.content?.model?.blocks ?? null,
-            metadata: { type: AV_EMBEDS },
+            metadata: {
+              assetId: parsedRoute.assetId,
+              language,
+              mediaId: parsedRoute.mediaId,
+              mediaDelimiter: parsedRoute.mediaDelimiter,
+              service,
+              type: AV_EMBEDS,
+              variant,
```

Comment:
Do service & variant need to live in metadata? Can't we get it from the service context?